CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+cmake_minimum_required(VERSION 3.6)
+project(Pokopomclion)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_BUILD_TYPE Debug)
+ADD_DEFINITIONS(-DUNICODE)
+ADD_DEFINITIONS(-D_UNICODE)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -m32 -D_UNICODE -DUNICODE")
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
Does this really mean it would export all sysmbols? Only those in Exports.def should be exported.